mtp_cashbook/templates/base.html

```diff
@@ -41,4 +41,13 @@
       {% endfor %}
   </ul>
   {% endif %}
+
+  <div class="subheader-item phase-banner">
+    <span class="phase-tag phase-tag-alpha">
+      ALPHA
```

Comment:
Also: lowercase text and style it to uppercase using CSS? (If memory serves, most screen readers will read this as “Ay Ell Pee Haitch Ay”?)